vip-rest-api.php

```diff
@@ -68,7 +68,18 @@ function wpcom_vip_verify_go_rest_api_request_authorization( $namespace, $auth_h
  * @param  string $namespace RESET API route's namespace
  * @return bool
  */
-function wpcom_vip_go_rest_api_request_allowed( $namespace ) {
+function wpcom_vip_go_rest_api_request_allowed( $namespace, $cap = 'do_not_allow' ) {
+	// First check basic auth
+	$basic_auth_user = wpcom_vip_basic_auth_user();
```

Comment:
Should we have a check to make sure that we're running over SSL?

Reply:
We effectively force all REST API requests to HTTPS by returning code 406 for non-secure requests.